examples/resources/cloudflare_access_group/resource.tf

```diff
@@ -22,3 +22,16 @@ resource "cloudflare_access_group" "test_group" {
     ip = [var.office_ip]
   }
 }
+
+# Allow members of an Azure Group. The ID is the group UUID (id) in Azure:
```

Comment:
i'll 👍 this but generally, we don't have kitchen sink style examples in our Terraform documentation. this should live in our developer documentation with an overview of how to use Azure as opposed to only in our Terraform implementation.

Reply:
got it, thank you